ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js

```diff
@@ -15,7 +15,8 @@ import { Text, Box } from '../../../component-library';
 const Player = ({ type, cbor, cancelQRHardwareSignRequest, toRead }) => {
   const t = useI18nContext();
   const urEncoder = useMemo(
-    () => new UREncoder(new UR(Buffer.from(cbor, 'hex'), type), 400),
+    //For NGRAVE ZERO support please keep to a maximum fragment size of 200
```

Comment:
```suggestion
    // For NGRAVE ZERO support please keep to a maximum fragment size of 200
```
To satisfy the linter!